app/src/main/java/com/infomaniak/mail/ui/MainActivity.kt

```diff
@@ -119,6 +125,7 @@ class MainActivity : ThemedActivity() {
 
     override fun onResume() {
         super.onResume()
+        localSettings.appLaunches++
```

Comment:
Maybe move it to `onStart()` or `onCreate()` ?

Reply:
OnCreate may not be enough, as one can just put his app in background but often uses it so it is never killed by the system
But onStart seems better indeed